packages/camlp5/camlp5.7.08/opam

```diff
@@ -13,7 +13,7 @@ doc: "https://camlp5.github.io/doc/html"
 install: [make "install"]
 synopsis: "Preprocessor-pretty-printer of OCaml"
 depends: [
-  "ocaml" {>= "4.02" & <= "4.08.0"}
+  "ocaml" {>= "4.02" & <= "4.08.1"}
```

Comment:
```suggestion
  "ocaml" {>= "4.02" & < "4.09.0"}
```
Why not just use strict comparaisons?

Reply:
In particular camlp5 does not support 4.08.2 would that version be released, each OCaml version does require special support upstream, see https://github.com/camlp5/camlp5/tree/master/ocaml_stuff